description.md

```diff
@@ -11,6 +11,25 @@ Each section details how a specific type of content is to be developed, stored a
 
 ### Reading
 
+The reading content type is the most common form of storing educational material (or information, in general).
+As the name suggests the content is developed to be read, therefore it comes in the form of text.
+
+To develop reading content we use the [Markdown syntax](https://www.markdownguide.org/basic-syntax) and the following guidelines:
+
+- the heading level of a title is given by the number of `#`s that precede it.
+For example, the title of a document is preceded by a single `#`, whereas the chapters are preceded by two.
+- each sentence is written on a new line.
+- paragraphs are separated by an empty line.
+- links should be used as hyperlinks by using the `[]()` syntax as in [this example](https://theuselessweb.com/).
+
+The above are enforced by automated linter checks.
+
+When developing such materials we recommend that:
+
+- the reading material is succint and concise, following the "les is more" principle.
+- the first sentence of each section describes what is the intended purpose of the section.
+- if the reading material can be expressed in a more visual manner, the latter is preffered.
```

Comment:
```suggestion
- the reading material be concise, following the "less is more" principle.
- the first sentence of each section describe the intended purpose of the section.
- if the reading material can be expressed in a more visual manner, the latter is preferred.
```

- succinct (written with an extra `c`, despite this spelling being incorrect in other languages, such as Romanian) == concise
- use the [subjunctive mood](https://www.scribbr.com/verbs/subjunctive-mood/). It's indeed a relic, but so are crocodiles.